include/spirv-tools/libspirv.h

```diff
@@ -425,9 +425,70 @@ typedef enum spv_binary_to_text_options_t {
   // Reorder blocks to match the structured control flow of SPIR-V to increase
   // readability.
   SPV_BINARY_TO_TEXT_OPTION_REORDER_BLOCKS = SPV_BIT(9),
+  // Add style annotations to the disassembly, which can be post-processed by
```

Comment:
This is an interesting scheme for passing in-band metadata. 

I guess today I learned about shift-in and shift-out ASCII characters, and I guess this is a good use of them.

It's (yet another) markup scheme.  I guess SGML / XML / HTML have generalized versions of that.